src/infrastructure/applications/console/config.ts

```diff
@@ -1,15 +1,24 @@
 import {PostgresConnectionOptions} from '@steroidsjs/typeorm/driver/postgres/PostgresConnectionOptions';
-import {join} from 'path';
+import * as path from 'path';
 import * as fs from 'node:fs';
 import baseConfig from '../base/config';
 import {IConsoleAppModuleConfig} from './IConsoleAppModuleConfig';
 import {EntityCodeGenerateCommand} from '../../commands/entity-generator/EntityCodeGenerateCommand';
 import {MigrateCommand} from '../../commands/MigrateCommand';
 import {CommandModule} from 'nestjs-command';
 
-const sourceRoot = join(process.cwd(), 'src'); // TODO Use from nest-cli.json configuration?
 const isMigrateCommand = !!(process.argv || []).find(arg => /^migrate/.exec(arg));
 
+// For deployment to use files in dist directory.
+const envRootDir = process.env.APP_ENVIRONMENT === 'dev' ? 'src' : 'dist';
```

Comment:
Поискал, и понял что у нас по-дефолту эта переменная окружения не устанавливается, поэтому добавил проверку на falsy
@PashakArt пойдет?

```suggestion
const envRootDir = !process.env.APP_ENVIRONMENT || process.env.APP_ENVIRONMENT === 'dev' ? 'src' : 'dist';
```

Reply:
смотрел в эту сторону, нашел такой код - https://github.com/steroids/nest/blob/de3a462cd835547f03351c65eb757ef36b792d2f/src/infrastructure/applications/BaseApplication.ts#L16C1-L16C1

Насколько понимаю, приложение не запуститься если переменная `APP_ENVIRONMENT` не указана.

Reply:
А, ты прав, да. Тогда моя правка не нужна